### PR TITLE
Update ScalaWebSockets.scala

### PR DESCRIPTION
**Pull request by bfaissal**
Added example on how to detect client disconnection when using akka-stream to handle Websockets

# Pull Request Checklist

* [ x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [ x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [ x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x ] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [ x] Have you added copyright headers to new files?
* [x ] Have you checked that both Scala and Java APIs are updated?
* [x ] Have you updated the documentation for both Scala and Java sections?
* [x ] Have you added tests for any changed functionality?

# Helpful things

## Fixes
Added example on how to detect client disconnection when using akka-stream to handle Websockets

## Purpose
The play-scala documentation whats missing an example on how to detect client diconnection when handling websocket using akka-stream.
What does this PR do?
Added example on how to detect client disconnection when using akka-stream to handle Websockets

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

File: documentation/manual/working/scalaGuide/main/async/code/ScalaWebSockets.scala [hunk 1]
```diff
@@ -321,6 +324,16 @@ object Samples {
       val out = Source.single("Hello!").concat(Source.maybe)
 
       Flow.fromSinkAndSource(in, out)
+      .watchTermination(){ (_, fut) =>
+      fut onComplete {
```
Comment (hepin1989) [inline documentation/manual/working/scalaGuide/main/async/code/ScalaWebSockets.scala]: better with `future. onComplete...`

Comment (bfaissal) [reply documentation/manual/working/scalaGuide/main/async/code/ScalaWebSockets.scala]: changed

**Comment by renatocaval** [comment]
needs forward port to master as this was done on branch 2.6.x

**Comment by bfaissal** [comment]
I can do that !

**Comment by marcospereira** [comment]
Thank you, @bfaissal.

Let us know when you need a new review here. :-)

Best.

**Comment by marcospereira** [comment]
Hey @bfaissal, are you still interested in this change? Anything we can do to help you here?

Best.

**Comment by bfaissal** [comment]
Hey Marcos,
Yes it's on my todo... it's just that I was in a rush lately.
Thanks for asking !

**Comment by mkurz** [comment]
@bfaissal Are you still interested to work on this pull request?

**Comment by dwijnand** [comment]
Probably should target the 2.7.x branch so it can then be forward-ported to the 2.8.x branch.

**Comment by dwijnand** [comment]
Yeah, needs to be rebased for that.

**Comment by Captain1653** [comment]
The PR looks like too old. Play 2.6 is unsupported. Is it worth to close it?

**Comment by octonato** [comment]
Yes, this is very old and seems that nobody is interesting to get this further. Also targeting a very old branch. 

I'm closing it. If someone feels in the mood of picking it again, please send a PR to `master` instead.

**Pull request closed without merging**